parts/linux/cloud-init/artifacts/ubuntu/cse_install_ubuntu.sh

```diff
@@ -18,7 +18,7 @@ installDeps() {
     aptmarkWALinuxAgent hold
     apt_get_update || exit $ERR_APT_UPDATE_TIMEOUT
     apt_get_dist_upgrade || exit $ERR_APT_DIST_UPGRADE_TIMEOUT
-    for apt_package in apache2-utils apt-transport-https blobfuse=1.3.7 ca-certificates ceph-common cgroup-lite cifs-utils conntrack cracklib-runtime ebtables ethtool fuse git glusterfs-client htop iftop init-system-helpers iotop iproute2 ipset iptables jq libpam-pwquality libpwquality-tools mount nfs-common pigz socat sysfsutils sysstat traceroute util-linux xz-utils zip; do
```

Comment:
Can we use a variable for the version so we only need to change the version number?

Reply:
write like this? but we still need to change version num in this `cse_install_ubuntu.sh` script, right?
```
BLOBFUSE_VERSION="1.3.8"
for apt_package in apache2-utils apt-transport-https blobfuse=${CONTAINERD_VERSION} ca-certificates
```

Reply:
Yes. but you will not see the diff in the long line.